example/lib/pages/components/switch_example.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+import 'package:zeta_example/widgets.dart';
+import 'package:zeta_flutter/zeta_flutter.dart';
+
+class SwitchExample extends StatefulWidget {
+  static const String name = 'Switch';
+
+  const SwitchExample({Key? key}) : super(key: key);
+
+  @override
+  State<SwitchExample> createState() => _SwitchExampleState();
+}
+
+class _SwitchExampleState extends State<SwitchExample> {
+  bool? isOnAndroid = false, isOnIOS = false;
+  bool isEnabledAndroid = true, isEnabledIOS = true;
+
+  @override
+  Widget build(BuildContext context) {
+    return ExampleScaffold(
+      name: 'Switch',
+      child: Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+          children: [
+            Row(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
+              children: [
+                Text('Android'),
+                ZetaSwitch.android(
+                  value: isOnAndroid,
+                  onChanged: isEnabledAndroid ? (value) => setState(() => isOnAndroid = value) : null,
```

Comment:
When toggling this it seems that the switch is changing dimensions by a few pixels because the 'Android' text shifts slightly. Is there any way we can prevent this from happening?

Reply:
I can see 'Android' moves when toggling between enabled/disabled because the size of the ZetaButton changes when different words are inside (Enable/Disable). But I can't see this happening when toggling the switch itself (On/Off). Could you please check once again.